### DISCUSSION TITLE
[Feature Request] Connect Terminal to Container

### DISCUSSION THREAD

**Discussion by jordan-wirth** [louislam/dockge#148] [Feature Request]
It would be useful to have a 'Terminal' button on each container in the list of a stack's containers, which would launch a web terminal for /bin/bash inside of a container after it has started. This is one of the only things I frequently use Portainer for.

**Comment by louislam** [comment]
I think it is already existing. Check Bash button in the container item.

**Reply by Zaid-maker** [reply]
Yes he's right
